system_files/desktop/shared/usr/share/ublue-os/just/custom.just

```diff
@@ -27,16 +27,7 @@ install-bazzite-arch:
 
 configure-waydroid:
   #!/usr/bin/env bash
-  cd ~
-  git clone https://github.com/casualsnek/waydroid_script
-  cd ~/waydroid_script
-  python -m venv venv
-  source venv/bin/activate
-  sudo pip install -r ~/waydroid_script/requirements.txt
-  sudo ~/waydroid_script/main.py
-  sudo pip uninstall -r ~/waydroid_script/requirements.txt -y
-  deactivate
-  rm -rf ~/waydroid_script
+  cd ~ && sudo rm -rf ~/waydroid_script/ && git clone https://github.com/casualsnek/waydroid_script && cd ~/waydroid_script && python -m venv venv && source venv/bin/activate && sudo pip install -r requirements.txt && sudo ~/waydroid_script/main.py && deactivate && sudo rm -rf ~/waydroid_script/
```

Comment:
Let's change this to not need cd if we're able, this is how I'd do it:

```
git clone https://github.com/casualsnek/waydroid_script.git --depth 1 /tmp/waydroid_script
python -m venv /tmp/waydroid_script/venv
source /tmp/waydroid_script/venv/bin/activate
sudo pip install -r /tmp/waydorid_script/requirements.txt
sudo /tmp/waydroid_script/main.py
deactivate
sudo rm -rf /tmp/waydroid_script
```

Reply:
I thought bazzite used to use a solution like this, however it didn't work. Just tried this, and it did? Although it also looks like waydroid_script has been updated, so maybe they fixed the issue we've been experiencing. Feel free to push the updated script you made.

Reply:
Changes made, thanks for testing!